### PR TITLE
add mamba recipe

### PR DESCRIPTION
**Pull request by wolfv**

### CODE REVIEW

File: recipes/mamba/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  url: https://github.com/wolfv/mamba/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - {{ compiler('cxx') }}
```
Comment (chrisburr) [inline recipes/mamba/meta.yaml]: The compiler should be in the `build` section of the requirements

File: recipes/mamba/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    - setuptools
+    - python
+    - pip
+    - pybind11 2.2.*
+    - conda
+    - libsolv
+  run:
+    - python
+    - conda
+    - vs2015_runtime  # [win]
```
Comment (chrisburr) [inline recipes/mamba/meta.yaml]: I think this shouldn't be needed

**Comment by wolfv** [comment]
@chrisburr thanks for taking a look! 

can you help me out on OS X? It's working locally, but somehow the linking doesn't seem to properly work on Azure...

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=23973

Locally, I had to add `-Wl,-rpath -Wl,/my/conda/prefix/lib` to add a dynamic search path for the library. Is there any recipe to steal this from?

File: recipes/mamba/meta.yaml (hunk 2 above)
Comment (wolfv) [reply recipes/mamba/meta.yaml]: gone!

**Comment by SylvainCorlay** [comment]
@wolfv You should be able to remive libsolv from the runtime requirements now that it is in host.

File: recipes/mamba/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
+{% set name = "mamba" %}
+{% set version = "0.0.1" %}
+{% set sha256 = "91787747592df337d79c97752f80557ab1cd67fd234e24764fc52b186df76f52" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```
Comment (chrisburr) [inline recipes/mamba/meta.yaml]: This line isn’t needed:
```suggestion
```

Comment (chrisburr) [inline recipes/mamba/meta.yaml]: This line isn’t needed:
```suggestion
```

**Pull request merged**